_source/logzio_collections/_log-sources/aws-cost-and-usage-report.md

```diff
@@ -18,6 +18,17 @@ shipping-tags:
   - aws
 order: 830
 ---
+
+<!-- tabContainer:start -->
+<div class="branching-container">
+
+* [Manual configuration](#manual-configuration)
```

Comment:
The name of this tab should be "Automated CloudFormation deployment" or something like that (it's not a configuration)